### ISSUE TITLE
Homey doesn't respond on 'learn command button'

### ISSUE THREAD

**Issue by hplos** [RWensveen/com.broadlink#30]
I can add my RM3 to the homey, but when I want to learn commands, nothing happens.

**Comment by hplos** [comment]
I uninstalled this app and now using IFTTT. Homey starts a IFTTT flow. This flow can communicate with the Broadlink cloud and sets the device I want to control with my RM mini 3 to the correct settings

**Comment by C0kedk** [comment]
Only the broadlink switch has ifttt support -right?

**Comment by jhlos** [comment]
No, I have 2 rm mini's and no switch / hub. The are connected to my wifi and IFTTT. My homey also is connected to IFTTT (only for my rm mini's) and thats working fine (a bit slow, but I can do everything I want)

**Comment by hplos** [comment]
I've got two rm mini's which are connected to wifi and the broadlink cloud (also for usage with the broadlink app) and further no broadlink hub / switch / whatever, only my homey, which also has the ifttt app. So my homey controlls the broadlinks by using ifttt
